prow/plugins.yaml

```diff
@@ -555,6 +555,9 @@ plugins:
   kubernetes-sigs/gcp-compute-persistent-disk-csi-driver:
   - trigger
 
+  kubernetes-sigs/sig-storage-local-static-provisioner:
+  - trigger
```

Comment:
Maybe add all the ones in the kubernetes org?

Reply:
+1
If you all agree I can update this pr to enable trigger by default.

Reply:
Or submit a separate PR, what do you think?

Reply:
@fejta: kubernetes-sigs should have those defaults already. trigger is the major one we don't enable automatically (I think there are token concerns)

Reply:
service account, etc. tokens? It seems reasonable not to enable trigger by default. I have created a PR to remove `preset-service-account: "true"`: https://github.com/kubernetes/test-infra/pull/10535.

Reply:
So this simply appends plugins that aren't already enabled at the k-sigs level? 👍